core/src/main/scala/com/banno/kafka/consumer/ConsumerOps.scala

```diff
@@ -363,17 +384,23 @@ case class ConsumerOps[F[_], K, V](consumer: ConsumerApi[F, K, V]) {
     * using auto-offset-commits, since it will not commit offsets for failed
     * records when the consumer is closed. The consumer must be configured to
     * disable offset auto-commits, i.e. `enable.auto.commit=false`.
+    *
+    * If keepAliveInterval is provided and finite, and topics have been manually
+    * assigned, then committed offsets will be periodically refreshed to prevent
+    * them from expiring for low traffic topics.
```

Comment:
> and topics have been manually assigned

Same as on `readProcessCommit` we should be able to remove this.

Reply:
See comments above :smile: